lib/User.rb

```diff
@@ -0,0 +1,211 @@
+require 'pry'
+require './Person'
+require './GroceryList'
+
+class User < Person 
+  attr_accessor :name, :age, :gender
+  def initialize(args = {})
+    super(args)
+      @spaghetti     = args[:spaghetti]
+    # @answers   = args[:answers]
+  end
+
+  def number_of_linefeeds(num)
+    num.times {puts}
+  end
+
+  def print_is_user_ernest_message(user)
+    puts name == "Ernest" ? "I knew it was you!" : "Sorry, you're not Ernest."
+  end
+
+  def when_is_user_is_75(user)
+    delta = (75 - age).abs
+    case age
+    when 0..74
+      puts "You'll be 75 in #{delta} years."
+    when 75
+      puts "You're 75!!!!"
+    when 76..150
+      puts "You turned 75 #{delta} years ago."
+    end
+  end
+
+  def get_grandparent_status?      
+    has_grandkids = gets.chomp.upcase
+    has_grandkids == "Y"
+  end
```

Comment:
Is this specific to users? I think it applies to all people. Put this guy in the Person class.